Language/Classes/Instance_Methods/Method_noSuchMethod/superclass_A01_t01.dart

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) 2023, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// @assertion It is a compile-time error if the name m is noSuchMethod
+/// forwarded in a concrete class C, and a superclass of C has an accessible
+/// concrete declaration of m which is not a noSuchMethod forwarder.
+///
+/// @description Checks that it is a compile-time error if the name `m` is
+/// `noSuchMethod` forwarded in a concrete class `C`, and a superclass of `C`
+/// has an accessible concrete declaration of `m` which is not a `noSuchMethod`
+/// forwarder.
```

Comment:
Perhaps mention that `m` is only noSuchMethod forwarded here if the inherited method has a signature which is not a correct override of the signature in the interface of the class here. Otherwise we'd just have an implemented method which is fine, and that's the one we'd invoke if we call `m()`. Something like ..

```suggestion
/// forwarder. Note that in this situation the inherited `m` must have a
/// signature which is not a correct override of the signature of `m`
/// in this class, otherwise `m` wouldn't be noSuchMethod forwarded.
```